website/docs/developer_guides/extension_points.md

```diff
@@ -356,7 +356,7 @@ Example: replaces about feature details widget for a particular track ID
 
 ```typescript
 pluginManager.addToExtensionPoint(
-  'Core-replaceAbout',
+  'Core-replaceWidget',
```

Comment:
this appears to remove replaceAbout, but replaceAbout is still a thing that exists :)

Reply:
`Core-replaceAbout` has its own docs on line 266 of this file, i think its docs were replicated in the `Core-replaceWidget` section by accident

Reply:
nice catch